web/src/linkifiers.ts

```diff
@@ -59,12 +65,14 @@ function python_to_js_linkifier(pattern, url) {
         // We have an error computing the generated regex syntax.
         // We'll ignore this linkifier for now, but log this
         // failure for debugging later.
-        blueslip.error("python_to_js_linkifier: " + error.message);
+        if (error instanceof SyntaxError) {
+            blueslip.error("python_to_js_linkifier: " + error.message);
+        }
```

Comment:
Why this `instanceof` check?

Reply:
(I think it's probably the case that the main error type we expect is a `SyntaxError` -- but other errors probably need to be handled similarly -- certainly we want them logged for investigation.)

Reply:
Thanks for the timely review. 
- Yes, you are right the other errors also need to be logged for investigation. I will fix this soon and push the changes when ready. 

- `SyntaxError` is the error that is thrown when the RegExp constructor fails due to any reason. [Link to MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/RegExp#exceptions). If the `instanceof` check is not done there's no way for TypeScript to know that `error` has a `message` field as well.

Reply:
While checking the code in the `try` block, I realized that the only possible error it could give is of type `SyntaxError`, this is because the only thing the code does is create a `RegExp` object, which can only throw a `SyntaxError`. 
Source : [Link to MDN Docs for RegExp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/RegExp#exceptions)
Therefore, there is no need to handle any other errors since they won't happen. However, I am open to discussion and could be wrong.

Reply:
I think we should have an `else throw error` block -- it's a bad practice to swallow exceptions that you don't expect (e.g. if we had a typo and one of the parameters were invalid!). I'll add that and merge.

Reply:
Thank you for clarifying that - I see it clearly now. It makes complete sense.